### PR TITLE
[Time Conductor] Realtime presets and history tracking

### LINKED ISSUE #3213: Time Conductor should include useful built-in time spans and history tracking
Screen Shot 2020-07-20 at 10 47 33 AM

In a similar fashion to a feature recently added for Fixed Timespan mode the Time Conductor's real-time mode should have useful pre-determined timespans available to the user via a menu. Additionally, we should watch, remember and make those available in the same way as well. Ideally, these are configurable per deployment.

### Timespans:
00:60:00 - 00:00:00
00:30:00 - 00:00:00
00:15:00 - 00:00:00
00:05:00 - 00:00:00
00:01:00 - 00:00:00

### PR DESCRIPTION
**Pull request by jvigliotta**
Added to the Time Conductor (while in realtime or local clock mode) presets (configurable upon deployment) as well as history tracking. Very similar in functionality to fixed time mode presets and history tracking.

Closes #3213

## Author Checklist
Changes address original issue? Yes
Unit tests included and/or updated with changes? N/A
Command line build passes? Yes
Changes have been smoke-tested? Yes
Testing instructions included? Yes

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -35,7 +35,13 @@
     </body>
     <script>
         const THIRTY_SECONDS = 30 * 1000;
-        const THIRTY_MINUTES = THIRTY_SECONDS * 60;
+        const ONE_MINUTE = THIRTY_SECONDS * 2;
+        const FIVE_MINUTES = ONE_MINUTE * 5;
+        const FIFTEEN_MINUTES = FIVE_MINUTES * 3;
+        const THIRTY_MINUTES = FIFTEEN_MINUTES * 2;
```
Comment (nikhilmandlik) [inline index.html:41]: 👍

**Review by nikhilmandlik — DISMISSED** [review]
LGTM

**Review by charlesh88 — DISMISSED** [review]
LGTM

File: src/plugins/timeConductor/ConductorHistory.vue [hunk 2]
```diff
@@ -196,11 +247,24 @@ export default {
             return records;
         },
         formatTime(time) {
+            let format = this.timeSystem.timeFormat;
+            let negativeOffset = false;
```
Comment (akhenry) [inline src/plugins/timeConductor/ConductorHistory.vue:251]: let's use the `is` prefix for booleans.

**Review by akhenry — APPROVED** [review]
Looks great! Just a very small comment.

File: src/plugins/timeConductor/ConductorHistory.vue (hunk 2 above)
Comment (jvigliotta) [reply src/plugins/timeConductor/ConductorHistory.vue]: updated! (apparently a while ago and I didn't let you know)

**Review by akhenry — APPROVED** [review]
### Reviewer Checklist

1. Changes appear to address issue? Y
2. Appropriate unit tests included? N/A - followup issue to address conductor coverage - https://github.com/nasa/openmct/issues/3244
3. Code style and in-line documentation are appropriate? Y
4. Commit messages meet standards? Y
5. Has associated issue been labelled `unverified`? (only applicable if this PR closes the issue)

**Pull request merged**